### ISSUE TITLE
Product indexes are not transferring from tmp to main index

### ISSUE THREAD

**Issue by Mobecls** [algolia/algoliasearch-magento#814]
Hi @JanPetr , it happens so that when we do full reindex, all our products are indexed to ...tmp index in algolia, but main index is kept untouched and no products updated...

that started happening after we updated to the latest version couple of days ago.

**Comment by JanPetr** [comment]
Hello @Mobecls,

can you check `algoliasearch_queue` table and see if the jobs `moveIndex` are correctly processed?

**Comment by Mobecls** [comment]
They are moving, but sometimes we get PID... we didn't find to what it is connected. And it is not only at moveIndex job, but for all jobs in Queue (fullReindex, indexproduct, moveIndex).

We are still investigating it.

**Comment by JanPetr** [comment]
ok, let me know if you'll manage to find out the root cause.

**Comment by JanPetr** [comment]
Hello @Mobecls,
since I didn't hear from you for a while I'm closing the issue.
Feel free to reopen if your problem persists.

**Comment by Mobecls** [comment]
Hello @JanPetr , 

we have the same issue on our staging server. 

Here is what happens:

1. I have enabled Queue,
2. in tmp index we already have 2900 products (so I would like just to transfer them to needed index)
3. I have run php shell/indexer.php --reindex algolia_search_indexer and it created in algoliasearch_queue table 3 pages of tasks. I have deleted all of them and left just two:

saveSettings - {"isFullProductReindex":true} 
moveProductsTmpIndex - {"store_id":"1"}

4. run php shell/indexer.php --reindex algolia_queue_runner (and verified that tasks disappeared from db). 

I expected, that all that will move products from tmp index (staging_default_products_tmp) to working index (staging_default_products) - but it doesn't... I still can see 2900 products in TMP index.


Disabling queue doesn't change anything.
